### PR TITLE
Tests README

### PR DESCRIPTION
**Pull request by intelmark**
Added information about how to add new tests to the
test Python file.

### CODE REVIEW

**Comment by 1480c1** [comment]
Right now reviewing, but be careful because newline != `\n` in markdown

To actually use a newline in a paragraph, you have to append a trailing `\`

**Comment by 1480c1** [comment]
look at <https://github.com/intelmark/SVT-HEVC/tree/test_documentation/Tests#Paths> for what I mean

**Comment by intelmark** [comment]
@1480c1 Thanks for reviewing.  I'm using a visual md editor (Typora ) and it looks correct there and in the browser.  Are there any places where a new line is lacking?

**Comment by 1480c1** [comment]
> @1480c1 Thanks for reviewing. I'm using a visual md editor (Typora ) and it looks correct there and in the browser. Are there any places where a new line is lacking?

It's missing in nearly everywhere due to the way that github merges the newlines into a paragraph unless there is a blank line in between, a new heading, or a trailing `\`

Be careful since GitHub has it's own markdown engine/parser and markdown visual editors will not all display the same way. (I'm assuming that Typora's engine parses `\n` as `\n`)

**Comment by intelmark** [comment]
Could you let me know which browser and OS you are viewing this file on?

thanks,
mark

**Comment by 1480c1** [comment]
Windows Chrome

**Comment by intelmark** [comment]
image

On my Windows Chrome (v75.0.3770.142), the file appears as in the attachment.  Is there something visually different from what you are seeing?

**Comment by 1480c1** [comment]
No, that's the same. I'm mainly talking about
image
Where it's one continuous paragraph

File: Tests/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+QP_VBR_MODE defines how the quantization parameter and variable bitrate parameters are specified in the tests:\
+0 = Both QP and VBR parameters are used\
+1 = Only QP parameter is used\
+2 = Only VBR parameter is used\
+	
+
+### Encoder Modes
+
+SPEED_ENC_MODES is a list variable that defines which encoding modes (0-12) are used in the tests. It defaults to [0,6,9]
+
```
Comment (tianjunwork) [inline Tests/README.md]: max mode is updated to 11

Comment (tianjunwork) [reply Tests/README.md]: please also review the change of test script in #263

**Pull request merged**